src/plugins/drawing/model/drawing-content.ts

```diff
@@ -49,7 +49,8 @@ export const DrawingContentModel = TileContentModel
     vectorType: types.maybe(types.enumeration<VectorType>("VectorType", Object.values(VectorType))),
     // is type.maybe to avoid need for migration
     currentStampIndex: types.maybe(types.number),
-    zoom: types.optional(types.number, 1)
+    zoom: types.optional(types.number, 1),
+    isNavigatorVisible : types.optional(types.boolean, true),
```

Comment:
Even though the tile navigator component is generic and could potentially be used by other tiles (including the Coordinate Grid which we'll be adding it to shortly), I only added this property and the related actions here in the drawing content model. I considered adding them to `TileContentModel` but I'm not sure that makes sense. There's very little in `TileContentModel` and I'm not sure we'll be adding the navigator option to more than two tile types. So maybe it's fine to only add the property and actions to tile models that support the navigator?

Reply:
One option would be to make a class in between, so that `DrawingContentModel` inherits from `NavigatableTile` which inherits from `TileContentModel`.  Then the navigator-related properties could be defined in `NavigatableTile` and Coordinate Grid could also inherit them.

Reply:
Thanks. That makes sense and I went with that option. I think there may be some remaining issues around naming, typing, and where exactly some things get defined, but it's not clear to me if we should try to get that perfected now or wait till we add navigator support in the Coordinate Grid. I tend to think it'd be good to wait, but let me know what you think.